packages/core/src/Stores/client-store.js

```diff
@@ -2718,6 +2720,28 @@ export default class ClientStore extends BaseStore {
         this.should_show_effortless_login_modal = should_show_effortless_login_modal;
     }
 
+    setPasskeysStatusToCookie(status) {
```

Comment:
i see there is no dependency with any observables for this function.
do we need this function in client store or shared package is fine?

Reply:
I did not get you, why do we need to add specific to passkey function to shared package?

Reply:
no i mean shared package becuase, this function is getting called from core package and accounts package.